cinema/models.py

```diff
@@ -85,9 +88,8 @@ def clean(self):
             (self.row, "row", "count_rows"),
             (self.seat, "seat", "count_seats_in_row"),
         ]:
-            count_attrs = getattr(
-                self.movie_session.cinema_hall, cinema_hall_attr_name
-            )
+            count_attrs = getattr(self.movie_session.cinema_hall,
+                                  cinema_hall_attr_name)
```

Comment:
The attributes `count_rows` and `count_seats_in_row` do not exist in the `CinemaHall` model. You should replace them with `rows` and `seats_in_row`, respectively, to match the existing attributes in the `CinemaHall` model.